src/Pages/ProductDetail/Components/AddReview/AddReview.js

```diff
@@ -0,0 +1,165 @@
+import React, { Component } from 'react';
+import './AddReview.scss';
+
+export default class AddReview extends Component {
+  constructor() {
+    super();
+    this.state = {
+      fileArr: [],
+      review: [],
+      reviewContents: '',
+      star: '5',
+    };
+  }
+
+  handleFileInput = e => {
+    const { fileArr } = this.state;
+    const { files } = e.target;
+    this.setState(
+      {
+        fileArr: [
+          ...fileArr,
+          {
+            id: fileArr.length + 1,
+            src: window.URL.createObjectURL(files[0]),
+          },
+        ],
+      },
+      () => {
+        // console.log(fileArr);
+      }
+    );
+  };
+
+  handleReviewValue = e => {
+    const { value, name } = e.target;
+    this.setState({
+      [name]: value,
+    });
+  };
+
+  deletePhoto = id => {
+    const { fileArr } = this.state;
+    this.setState({
+      fileArr: fileArr.filter(file => file.id !== id),
+    });
+  };
+
+  addReview = () => {
+    const { reviewContents, review, star } = this.state;
+    if (!reviewContents) {
+      alert('리뷰 내용을 입력해주세요.');
+    } else {
+      this.setState(
+        {
+          review: [
+            ...review,
+            {
+              star,
+              reviewContents,
+            },
+          ],
+        },
+        () => {
+          // console.log(review);
+        }
+      );
+    }
+  };
+
+  render() {
+    const { fileArr } = this.state;
+    const { deletePhoto, handleReviewValue, handleFileInput, addReview } = this;
+    return (
+      <div className="reviewWrap">
+        <p>후기</p>
+        <div className="reviewHeader">
+          <span>REVIEW&emsp;| </span>
+          <span className="reviewDesc">
+            리뷰의 글자수, 구매도움수, 사진 여부, 최신작성일 등을 바탕으로 여백
+            0100이 추천하는 리뷰를 정합니다.
+          </span>
+        </div>
+        <div className="textareaWrap">
+          <textarea
+            onChange={e => handleReviewValue(e)}
+            name="reviewContents"
+          ></textarea>
```

Comment:
```suggestion
        <div className="textareaWrap">
          <textarea
            onChange={handleReviewValue}
            name="reviewContents"
          />
```
- 셀프 클로징 태그 사용할 수 있는 부분은 적용해주세요
- 즉시실행함수로 써야하는 부분만 사용해주세요. 지금은 불필요해보입니다! 다른 부분도 찾아서 고쳐주세요.

Reply:
넵 수정하겠습니다~